src/ascii.rs

```diff
@@ -28,9 +28,9 @@ where
     pub async fn get<K: AsRef<[u8]>>(&mut self, key: K) -> Result<Vec<u8>, Error> {
         // Send command
         let writer = self.io.get_mut();
-        writer.write_all(b"get ").await?;
-        writer.write_all(key.as_ref()).await?;
-        writer.write_all(b"\r\n").await?;
+        writer
```

Comment:
Looks like a nice change to me. Slice concat (for small bytes) is much cheaper chan create two more futures.